addons/html_builder/static/src/plugins/google_maps_option/google_maps_api_key_dialog.js

```diff
@@ -0,0 +1,51 @@
+import { _t } from "@web/core/l10n/translation";
```

Comment:
i'm not really reviewing this, but i have a quick remark: all this code should go into `html_builder/static/src/website_builder`. because it will be moved later in the website addon.

Reply:
Ah ok! My whole `google_maps_option` folder?

Reply:
Understood, done. What about the tests though?